tests/clients/boto3/test_boto3_lambda.py

```diff
@@ -2,74 +2,256 @@
 # (c) Copyright Instana Inc. 2020
 
 from __future__ import absolute_import
+import unittest
+import json
 
-import os
 import boto3
-import pytest
-
 # TODO: Remove branching when we drop support for Python 3.7
-import sys
-if sys.version_info >= (3, 8):
+from sys import version_info
+if version_info >= (3, 8):
   from moto import mock_aws
 else:
```

Comment:
I think, this `else` branch should be superfluous, since the only TC in this file is skipped anyway.